be/test/CMakeLists.txt

```diff
@@ -279,3 +279,7 @@ TARGET_LINK_LIBRARIES(bytes_test
         gtest
         tcmalloc
         )
+
+add_executable(starrocks_column_to_arrow_test ./util/arrow/starrocks_column_to_arrow_test.cpp)
+TARGET_LINK_LIBRARIES(starrocks_column_to_arrow_test ${TEST_LINK_LIBS})
+SET_TARGET_PROPERTIES(starrocks_column_to_arrow_test PROPERTIES COMPILE_FLAGS "-fno-access-control")
```

Comment:
is that compiler flag for testing?

Reply:
building a huge starrocks_test file takes so long time